swift/test/Slice/escape/Clash.ice

```diff
@@ -1,57 +1,55 @@
 // Copyright (c) ZeroC, Inc.
 module Clash
 {
+    interface Intf
+    {
+        void context();
+        void current();
+        void response();
+        void upCast();
+        void typeId();
+        void del();
+        void cookie();
+        void sync();
+        void inS();
+        void istr();
 
-interface Intf
-{
-    void context();
-    void current();
-    void response();
-    void upCast();
-    void typeId();
-    void del();
-    void cookie();
-    void sync();
-    void inS();
-    void istr();
-
-    void op(string context, string current, string response, string ex, string sent, string cookie,
+        void op(string context, string current, string response, string ex, string sent, string cookie,
             string sync, string result, string istr, string ostr, optional(1) string proxy);
-    void opOut(out string context, out string current, out string response, out string ex,
-               out string sent, out string cookie, out string sync, out string result, out string istr,
-               out string ostr, out optional(1) string proxy);
-}
-
-class Cls
-{
-    Intf* s;
-    string context;
-    int current;
-    short response;
-    string upCast;
-    int typeId;
-    short del;
-    optional(1) short cookie;
-    string ex;
-    int result;
-    string istr;
-    string ostr;
-    string inS;
-    string proxy;
-}
+            void opOut(out string context, out string current, out string response, out string ex,
+                out string sent, out string cookie, out string sync, out string result, out string istr,
+                out string ostr, out optional(1) string proxy);
+            }
```

Comment:
While the script is correctly indenting for '(' characters,
it seems like it isn't dedenting (the opposite of indenting) after it hits ')' here,

causing it permanently gain an indentation level for the rest of the file.